e2e/app/component/modal.ts

```diff
@@ -11,7 +11,7 @@ import * as fp from 'lodash/fp';
 import {waitWhileLoading} from 'utils/waits-utils';
 
 const Selector = {
-  defaultDialog: '//*[@role="dialog"]',
+  defaultDialog: '//*[@id="popup-root"]//*[@role="dialog" and contains(@class, "after-open")]',
```

Comment:
Puppeteer `waitForXpath` parameter `{visible: true}` can't be trusted to verify popup is open and visible. So I'm relying on class attribute to make sure visible. Similarly,  `menu` dropdown xpath selectors are updated.